src/modules/design/diagram-detail/diagram-detail.html

```diff
@@ -104,5 +104,51 @@ <h3 class="process-start-event-modal-dialog__header-headline">Select initial Sta
     </div>
     <div class="modal-backdrop fade in"></div>
   </div>
-    <!-- #endregion -->
+  <!-- #endregion -->
+
+  <!-- #region Save Modal for Process Start -->
+  <div show.bind="showStartWithOptionsModal">
+    <div class="modal show show-modal" tabindex="-1" role="dialog">
+      <div class="modal-dialog save-modal-dialog" role="document">
+        <div class="modal-content">
+          <div class="modal-header">
+            <h4 class="modal-title">Start Options</h4>
+          </div>
+          <div class="modal-body">
+            <form>
+              <div class="form-group">
+                <label for="correlationId"><b>Custom Correlation ID</b></label>
+                <input class="form-control input-field" class.bind="hasValidationError ? 'wrong-input' : ''" id="correlationId" aria-describedby="correlationIdHelp" placeholder="Correlation ID" type="text" value.bind="customCorrelationId">
+                <ul if.bind="hasValidationError" class="correlation-id-error__container">
+                  <li class="error-list__container">Your diagram contains at least one invalid-character
+                    Allowed characters: [a-z, 0-9, -, _, ä, ü, ö, ß, .]
```

Comment:
IMO 'Valid' or 'Supported' sounds a lot better

```suggestion
                    Valid characters: [a-z, 0-9, -, _, ä, ü, ö, ß, .]
```

```suggestion
                    Supported characters: [a-z, 0-9, -, _, ä, ü, ö, ß, .]
```